lib/internal/Magento/Framework/View/Result/Layout.php

```diff
@@ -7,6 +7,7 @@
 namespace Magento\Framework\View\Result;
 
 use Magento\Framework;
+use Magento\Framework\App\Response\HttpInterface as HttpResponseInterface;
```

Comment:
Do you see a reason not to change signature of `\Magento\Framework\Controller\ResultInterface::renderResult`? `\Magento\Framework\Controller\AbstractResult::renderResult` has the same issue with non-interface methods usage.